website/docs/d/dns_domain_registration.html.markdown.orig

```diff
@@ -0,0 +1,60 @@
+---

```

Comment:
The file has some conflcts...

Reply:
Junk file deleted. Left over from merging conflicts.